src/main/java/com/zipcodewilmington/looplabs/StringDuplicateDeleter.java

```diff
@@ -1,8 +1,92 @@
 package com.zipcodewilmington.looplabs;
 
+import java.util.Arrays;
 /**
  * Created by leon on 1/28/18.
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class StringDuplicateDeleter extends DuplicateDeleter<String> {
```

Comment:
looks like this class is mostly the same as Integer deleter. My feedback is pretty much the same.